<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Suivie_medical</class>
 <widget class="QDialog" name="Suivie_medical">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>suivi medical</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>1011</width>
     <height>661</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Ajouter suivi medical</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>401</width>
       <height>401</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajouter</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>poid</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>taille</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="_id">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="_poid">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="_taille">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>280</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajouter">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>360</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="_date">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>date</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>tension</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="_tenstion">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QTableView" name="tableView_aff">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>581</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>440</y>
       <width>501</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Suprime</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>ID :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_idS">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_S">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Suprime</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Afficher</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Modifier</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Supprime</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QLabel" name="label_p">
   <property name="geometry">
    <rect>
     <x>-470</x>
     <y>-180</y>
     <width>1781</width>
     <height>1181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_R">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>720</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Q">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>730</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <zorder>label_p</zorder>
  <zorder>tabWidget</zorder>
  <zorder>pushButton_R</zorder>
  <zorder>pushButton_Q</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
